madmom/ml/hmm.pyx

```diff
@@ -99,6 +99,52 @@ class TransitionModel(object):
         return np.log(self.probabilities)
 
     @staticmethod
+    def make_dense(states, pointers, probabilities):
+        """
```

Comment:
This does not relate to this PR directly, but I'm not sure if `make_dense` and `make_sparse` are good names here, because imho they just convert to/from different sparse representations of the transition matrix (CCO to CSR, see https://en.wikipedia.org/wiki/Sparse_matrix)...

Reply:
Yes, this is something which bothered me as well, especially since it doesn't become dense. How about compress/de-compress instead? We can deprecate `make_sparse` then -- although I don't think a lot of people use it, since this is a rather low-level routine. We could probably even prefix it with _.

Reply:
I will still merge this PR as it is now, since there exists already a `from_dense` method, so this should be addressed jointly. Created issue #335.